wandb/integration/kfp/wandb_logging.py

```diff
@@ -40,6 +41,10 @@ def isinstance_namedtuple(x):
     def get_iframe_html(run):
         return f'<iframe src="{run.url}?kfp=true" style="border:none;width:100%;height:100%;min-width:900px;min-height:600px;"></iframe>'
 
+    def get_link_back_to_kubeflow():
+        kubeflow_base_path = os.getenv("WANDB_KUBEFLOW_URL")
```

Comment:
Why are these lines not hit in tests, shouldn't they be?

Reply:
I'm not sure how to test these because they are inner functions.

AFAIK, these functions need to be in the decorator factory to work with kfp.